src/dotnet/Chat.UI.Blazor/Components/ChatMessageEditor/chat-message-editor.ts

```diff
@@ -132,6 +132,16 @@ export class ChatMessageEditor {
         this.changeMode();
     }
 
+    public clearAttachments() {
+        const attachments = this.attachments;
+        this.attachments.clear();
+        attachments.forEach(a => {
```

Comment:
Attachments will be empty after calling clear method, URL.revokeObjectURL will never be invoked.